src/gwas.jl

```diff
@@ -298,6 +323,107 @@ function iht_path(
     return betas
 end
 
+"""
+    iht_path(x::BEDFile, y, path)
```

Comment:
These comments should be updated to describe your new code

Reply:
done